### ISSUE TITLE
create-abell-app not working in windows

### ISSUE THREAD

**Issue by PiyushSuthar** [abelljs/create-abell-app#3]
## Issue
I have been trying to create an Abell project. But after running 
```sh
npx create-abell-app my-app --template abelljs/abell-starter-portfolio
```
It gets stuck there. I've been waiting for about 10 minutes, still nothing shows up.

![image](https://user-images.githubusercontent.com/32237558/93850508-e17f5300-fccb-11ea-98df-7b3712485ddd.png)

For a moment I thought that it might be an internet issue, so I connected to another WIFI network. But still the same issue... 

#### OS: Windows 10

**Comment by PiyushSuthar** [comment]
Hi @saurabhdaware , I cloned the repo locally and tried tweaking some code.

Here's the solution.

### I removed/commented the execOptions and it worked.

![image](https://user-images.githubusercontent.com/32237558/93859087-c6680f80-fcda-11ea-8d12-a71afe97a84b.png)


![image](https://user-images.githubusercontent.com/32237558/93859024-b5b79980-fcda-11ea-8ab9-b894998d963d.png)

**Comment by saurabhdaware** [comment]
Hey that's great! Would you like to send a Pull Request? If you need, I can help with the process.

**Comment by PiyushSuthar** [comment]
Yeah! but I'm not sure if it's working on any other OS.

Why did you add execOptions?

**Comment by saurabhdaware** [comment]
execOptions apparently add extra logs to git clone and npm install.

So we get these logs because of stdio option.
![image](https://user-images.githubusercontent.com/30949385/93860996-7b033080-fcdd-11ea-886e-252ecea19c85.png)

without stdio, we're not suppose to get them.

I think that is fine since working is more important right now. We can figure out something for it later.

**Comment by PiyushSuthar** [comment]
I just figured out that the problem is only with `git clone ....` not with `npm install`.

We can remove execOptions from 
```js
execSync(`git clone ${template} ${projectName}`, execOptions)
```

And keep execOption for 
```js
execSync(`cd ${projectName} && ${installCommand}`, execOptions)
```
This will log the npm's logs...

**Comment by saurabhdaware** [comment]
Yup that's great! that should work

**Comment by saurabhdaware** [comment]
Fixed in #5
